pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -101,3 +107,14 @@ func addDefaulter(mgr kube_ctrl.Manager, gvk kube_schema.GroupVersionKind, facto
 func generateDefaulterPath(gvk kube_schema.GroupVersionKind) string {
 	return fmt.Sprintf("/default-%s-%s-%s", strings.Replace(gvk.Group, ".", "-", -1), gvk.Version, strings.ToLower(gvk.Kind))
 }
+
+func addValidators(mgr kube_ctrl.Manager) error {
+	handler, err := k8s_webhooks.NewValidatingWebhook(k8s_resources.DefaultConverter(), core_registry.Global(), k8s_registry.Global())
+	if err != nil {
+		return err
+	}
+	path := "/validate-kuma-resource"
```

Comment:
For consistency with the defaulter, let's use a path `/validate-kuma-io-v1alpha1`